addons/packages/pinniped/0.12.1/fixtures/values/mc-oidc-updatestrategyoverlay-v1_5_0.yaml

```diff
@@ -0,0 +1,18 @@
+#@data/values
+#@overlay/match-child-defaults missing_ok=True
+---
+#! Notes:
+#! - deployments (dex, supervisor, concierge) should honor both the updateStrategy as well as the nodeSelector
+#! - post-deploy job should honor the nodeSelector
+deployment:
+  updateStrategy: RollingUpdate #! RollingUpdate is only option that will trigger this in the deployment
+  rollingUpdate:
+    maxUnavailable: 1111
+    maxSurge: 9999
+nodeSelector:
+    race: "wood elf"
```

Comment:
praise: i shoulda set this yesterday, but this test fixture (and the other one) bring me joy :)

Reply:
hehe.